apps/devhub/views.py

```diff
@@ -1196,6 +1197,19 @@ def version_add(request, addon_id, addon):
                 form.cleaned_data['nomination_type']):
             addon.update(status=form.cleaned_data['nomination_type'])
         url = reverse('devhub.versions.edit', args=[addon.slug, str(v.id)])
+
+        if not addon.is_listed:  # Not listed? Do we need to sign?
+            # We need to get to the file to check its validation.
+            # We're submitting a new version here, so it has only one file.
+            file_ = v.files.get()
```

Comment:
I don't know if you're doing it but on zamboni we've tried to clean up single character variable names as we go.